src/www/ui/api/documentation/openapi.yaml

```diff
@@ -642,6 +642,20 @@ paths:
           in: header
           schema:
             type: string
+        - name: page
+          description: Page number (starts from 1)
+          required: false
+          in: header
+          schema:
+            type: integer
+            minimum: 1
+        - name: pageSize
```

Comment:
Consider call it `limit` as done in other endpoints

https://github.com/fossology/fossology/blob/a3c93461b11aefc5a6b5d0ceb4a4a34ed8621cee/src/www/ui/api/documentation/openapi.yaml#L772-L777

Reply:
Also, will be nice (not required) if you can expose total number of pages

https://github.com/fossology/fossology/blob/a3c93461b11aefc5a6b5d0ceb4a4a34ed8621cee/src/www/ui/api/documentation/openapi.yaml#L794-L797